src/traces/scattersmith/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+    moduleType: 'trace',
+    name: 'scattersmith',
+    basePlotModule: require('../../plots/smith'),
+    categories: ['polar', 'symbols', 'showLegend', 'scatter-like'],
```

Comment:
This `polar` category does not look correct.

Reply:
See 6258e635a362ad7b884a9fd1c8a5674763e7a02d.